source/develop/release-management/features/virt/zerocopy-migrations.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Zero-copy migrations
+category: feature
+authors: mzamazal
+---
+
+# Zero-copy migrations
+
+## Summary
+
+This feature adds support for zero-copy migrations.
+
+## Owner
+
+*   Name: Milan Zamazal
+*   Email: mzamazal@redhat.com
+
+## Description
+
+Zero-copy migrations further improve migration speed and may allow migrating large VMs (>1 TB RAM).
+
+Zero-copy migrations can be enabled by passing ``VIR_MIGRATE_ZEROCOPY`` flag to [virDomainMigrateToURI3](https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainMigrateToURI3) libvirt call.
+
+## Prerequisites
+
+Nothing special, just up-to-date libvirt and QEMU.
+
+## Limitations
+
+Current implementation of zero-copy migration in QEMU has some restrictions:
+
+- It can be used only with [parallel migrations](parallel-migrations.html) and thus the same limitations as for parallel migrations apply.
+
+- It cannot be used with migration encryption.
```

Comment:
and parallel migrations alone do work with encryption?

Reply:
Yes.